src/analytics/event.ts

```diff
@@ -78,9 +79,14 @@ export const event = {
   nftOffersSelectedSortCriterion: 'Selected NFT Offers Sort Criterion',
   nftOffersAcceptedOffer: 'Accepted NFT Offer',
 
+  nftMintsOpenedSheet: 'Opened NFT Mint Sheet',
+  nftMintsMintingNFT: 'Minting NFT',
+  nftMintsMintedNFT: 'Minted NFT',
```

Comment:
not a big deal but i think the naming convention should match the existing ones that i created for mints (see line 93 and below). could be confusing for our amplitude users